srcpkgs/libchewing/template

```diff
@@ -0,0 +1,35 @@
+# Template file for 'libchewing'
+pkgname=libchewing
+version=0.5.1
+revision=1
+#wrksrc=
+#create_wrksrc=yes
+#only_for_archs="i686 x86_64"
+build_style=cmake
+#configure_args=""
+#make_build_args=""
+#make_install_args=""
+#conf_files=""
+#make_dirs="/var/log/dir 0755 root root"
+hostmakedepends="autoconf automake libtool clang gcc"
+makedepends="sqlite-devel"
+depends="sqlite"
```

Comment:
Remove

Reply:
ping

Reply:
All the changes have been made in the most recent commit, are you saying this shouldn't depend on sqlite?